components/MonitoringCommentCard.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div class="MonitoringComment">
+    <div class="MonitoringComment-heading">
+      <h3 class="MonitoringComment-title">
+        {{ $t('感染状況・医療提供体制の分析') }}
+      </h3>
+    </div>
+    <div class="MonitoringComment-description">
+      <p>
+        {{
+          $t(
+            '東京都では、都内の「感染状況」と「医療提供体制」を2つの柱として7つのモニタリング項目を設定しています。'
+          )
+        }}
+        {{
+          $t(
+            '毎週、専門家による状況分析を項目ごとに行い、都内の感染状況と医療提供体制それぞれについて「総括コメント」として3段階で評価します。'
```

Comment:
すみません！テキスト修正を発見しました。
```suggestion
            '毎週、専門家による状況分析を項目ごとに行い、都内の感染状況と医療提供体制それぞれについて「総括コメント」として4段階で評価します。'
```

Reply:
こちらコミットしますね。